README.md

```diff
@@ -434,6 +434,8 @@ We have a public [logger](https://kibana.electricitymap.org/app/kibana#/discover
 
 To get started, [clone](https://help.github.com/articles/cloning-a-repository/) or [fork](https://help.github.com/articles/fork-a-repo/) the repository, and install [Docker](https://docs.docker.com/engine/installation/).
 
+If you want to run python code outside the docker environment, for example when testing a new parser, please use python 3.6 .
```

Comment:
I think the instructions already present in a later section are enough. I +1 the idea of letting know the contributor that the docker things are not necessary to test the parser, hence this suggestion instead : 
```suggestion
Note : to test parsers, head directly to [Testing parsers locally](#testing-parsers-locally)
```

Reply:
@ajoga I couldn't find any reference to the version of python (3.6) that we use though. Is that listed somewhere later on?

Reply:
I couldn't either. I'm unsure on what's best to offer to convey this information.